src/tests/ilverify/TestDataLoader.cs

```diff
@@ -176,20 +176,37 @@ private static TheoryData<TestCase> GetTestMethodsFromDll(Func<string[], MethodD
                     var method = (EcmaMethod)testModule.GetMethod(methodHandle);
                     var methodName = method.Name;
 
-                    if (!String.IsNullOrEmpty(methodName) && methodName.Contains("_"))
+                    if (methodName?.Contains("_") == false)
+                        continue;
+
+                    var index = methodName.LastIndexOf("_Valid");
```

Comment:
This will perform culture-specific comparison. Not something we want to do here.

Reply:
Ok, all test names are in ASCII range and `_Valid` is the documented token that must be present in the method under test. I don't think it will break anything, but will put OrdinalIgnoreCase.

Reply:
I would make it Ordinal. No need to ignore case mismatches here.